dali/operators/math/normalize/normalize.cc

```diff
@@ -192,7 +206,11 @@ void Normalize<CPUBackend>::RunTyped(HostWorkspace &ws) {
   }
 
   if (has_scalar_stddev_) {
-    scalar_inv_stddev = scale_ / spec_.GetArgument<float>("stddev");
+    float scalar_stddev = spec_.GetArgument<float>("stddev");
+    if (epsilon_)
+      scalar_inv_stddev = scale_ * rsqrt(scalar_stddev*scalar_stddev + epsilon_);
```

Comment:
It took me too long to connect the fact that we're adding epsilon to `Variance` and we're handling the `Standard deviation` here, hence the square and square root. Can you maybe add a comment that would make it more obvious here, the docs for epsilon is quite distant.

Reply:
Done.